payu/experiment.py

```diff
@@ -375,6 +376,18 @@ def set_output_paths(self):
         for model in self.models:
             model.set_model_output_paths()
 
+    def set_environment_vars(self):
```

Comment:
While trying to understand some parts of payu I saw there's an existing function `set_env_vars` in `cli.py` with a similar name. I'm wondering whether we should use a different name here to avoid the two getting mixed up. If the exported variables will only be used by userscripts, would something like `set_userscript_env_vars` be ok?

Reply:
Yeah I think that's a great idea - I've added that in that change